src/LesserPhp/Block.php

```diff
@@ -0,0 +1,124 @@
+<?php
+namespace LesserPhp;
+
+/**
+ * lesserphp
+ * https://www.maswaba.de/lesserphp
+ *
+ * LESS CSS compiler, adapted from http://lesscss.org
+ *
+ * Copyright 2013, Leaf Corcoran <leafot@gmail.com>
+ * Copyright 2016, Marcus Schwarz <github@maswaba.de>
+ * Copyright 2017, Stefan Pöhner <github@poe-php.de>
+ * Licensed under MIT or GPLv3, see LICENSE
+ *
+ * @package LesserPhp
+ */
+
+class Block
```

Comment:
BlockInterface will not help here. This is for autocomplete, otherwise just Block would do.